dandi/tests/test_models.py

```diff
@@ -154,3 +166,192 @@ def test_autogenerated_titles():
     assert schema["title"] == "Asset Meta"
     assert schema["properties"]["schemaVersion"]["title"] == "Schema Version"
     assert schema["definitions"]["PropertyValue"]["title"] == "Property Value"
+
+
+def datacite_post(datacite, doi):
+    """
+    posting the datacite object and returning the status codes
+    of requests.post and requests.get
+    """
+
+    # removing doi in case it exists
+    _clean_doi(doi)
+
+    # checking f I'm able to create doi
+    rp = requests.post(
+        "https://api.test.datacite.org/dois",
+        json=datacite,
+        headers={"Content-Type": "application/vnd.api+json"},
+        auth=("DARTLIB.DANDI", os.environ["DATACITE_DEV_PASSWORD"]),
+    )
+
+    # checking if i'm able to get the url
+    rg = requests.get(
+        url=f"https://api.test.datacite.org/dois/{doi.replace('/','%2F')}/activities"
```

Comment:
why activities and not compare to the posted metadata?

Reply:
also we shouldn't have to do the `doi.replace`

Reply:
indeed, the replacement is not needed. But the form with "activities" I took from datacite examples and it doesn't work without